examples/example.py

```diff
@@ -1,7 +1,18 @@
-import markdown
+import markdown, sys
+
+extensions = ['ditaa', 'graphviz', 'plantuml']
+
+try:
+    extension  = sys.argv[1]
+    if extension in extensions:
+        fin = open("%s.md" % extension, 'r')
+        txt = fin.read()
+        fin.close()
+        md = markdown.Markdown(extensions=[extension])
+        print md.convert(txt)
+    else:
+        print "Extension(%s) do not exist" % extension
+except:
```

Comment:
Even though this is just example code, we probably shouldn't use a bare except (see quote below).

```diff
-except:
+except Exception:
```

>A bare except: clause will catch SystemExit and KeyboardInterrupt exceptions, making it harder to interrupt a program with Control-C, and can disguise other problems. If you want to catch all exceptions that signal program errors, use except Exception: (bare except is equivalent to except BaseException: )